frontend/src/components/Layouts/HomePageComponent/components/CategoryOutput/CategoryOutput.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from 'react';
+import axiosInstance from '../../../../../helpers/axiosInstance';
+import styles from './CategoryOutput.module.scss';
+import Card from '../../../../common/Card';
+import { ICONS } from '../../../../ui-kit/icons';
+import { Link } from 'react-router-dom';
+import IconButton from '../../../../ui-kit/components/IconButton';
+
+const CategoryOutput = ({ api, link }) => {
+  const [data, setData] = useState();
+  useEffect(() => {
+    axiosInstance
+      .get(api)
+      .then((response) => {
+        setData(response.data.content);
+      })
+      .catch((error) => {
+        console.error(error);
+      });
+  }, [api]);
+  return (
+    <div className={styles.categoryOutput}>
+      <div className={styles.cardOutput}>
+        {data &&
```

Comment:
There may be an empty array, maybe it's better to use a ternary expression and if the array is empty, output something else?